api/redis/processReminders.js

```diff
@@ -0,0 +1,44 @@
+const reminderStore = require('./client.js').reminders;
+const bot = require('../bot.js');
+
+const getTimestamps = (array) => {
+  const timestamps = [];
+
+  array.forEach((timestamp) => {
+    if (timestamp < Date.now()) {
+      timestamps.push(timestamp);
+    }
+  });
+  return timestamps;
+};
+
+const sendReminders = (remindersArray, timestamp) => {
+  remindersArray.forEach((reminder) => {
+    bot.fetchUser(reminder.target).then((user) => {
+      user.send(reminder.content);
+    });
+  });
+
+  reminderStore.del(timestamp);
+};
+
+const getReminders = (timestampsArray) => {
+  timestampsArray.forEach(timestamp => reminderStore.getAsync(timestamp)
+  .then((remindersToSend) => {
+    sendReminders(JSON.parse(remindersToSend), timestamp);
+  }));
+};
+
+const processReminders = () => {
+  let expiredTimestamps;
```

Comment:
dunno if we need to set this

Reply:
Keeping this variable for readability. Otherwise, `getReminders(getTimestamps(data[1]))` is incomprehensible.

Reply:
how about a line break?
```
getReminders(
  getTimestamps(data[1])
)
```